### ISSUE TITLE
Add url to Sucess button

### ISSUE THREAD

**Issue by Alex2k15** [makeusabrew/bootbox#466]
Idk how to add a url to the Sucess button for my popup. Also my other button won't close

Check it out: icebear.cf

**Comment by Alex2k15** [comment]
anyone?

**Comment by tarlepp** [comment]
Url to success button? You mean that you want to redirect user to another url when he/she clicks the button? If that is the case you could easily do that in your callback function and just do something like ```window.location = 'http://google.com';```

Could you provide simple plunker example of your code, so that anything else is not messing up bootbox functionality.

**Comment by Alex2k15** [comment]
Here is my source

  bootbox.dialog({
  message: "I am a custom dialog",
  title: "Custom title",
  onEscape: function() {},
  buttons: {
    danger: {
      label: "Danger!",
      className: "btn-danger",
      callback: function() {
        Example.show("uh oh, look out!");
      }
    },
    main: {
      label: "Click ME!",
      className: "btn-primary",
      callback: function() {
        Example.show("Primary button");
       }
      }
    }
});

Haven't done much to it

**Comment by tarlepp** [comment]
And what is that ```Example.show()```?

**Comment by tarlepp** [comment]
Oh, now I get it. Seems like you have multiple issues on your site structure and actual Javascript. Look at your browser console to see those.

![image](https://cloud.githubusercontent.com/assets/595561/10300316/7890d8dc-6bfd-11e5-891b-e2afb4ff5839.png)

**Comment by tarlepp** [comment]
So basically you have following problems on your site:
1) You need to load jquery _before_ bootstrap
2) Something fishy is going on on that ```Example``` function

tl;dr none of these are really bootstrap  problems. 

I made a simple example for you how to redirect user to another url when he/she clicks the buttons on that dialog http://plnkr.co/edit/WisXnGdNQVtbg0QLY6UJ?p=preview

Closing this one now.

**Comment by Alex2k15** [comment]
Thank you :dancer:
